pkg/app/web/src/components/application-counts/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, memo } from "react";
+import { makeStyles } from "@material-ui/core";
+import { useSelector } from "react-redux";
+import { AppState } from "../../modules";
+import { ApplicationCount } from "../application-count";
+import {
+  ApplicationActiveStatus,
+  ApplicationKind,
+} from "../../modules/applications";
+import { APPLICATION_ACTIVE_STATUS_NAME } from "../../constants/application-active-status";
+import { APPLICATION_KIND_BY_NAME } from "../../constants/application-kind";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(2),
+    display: "flex",
+    justifyContent: "center",
+  },
+  count: {
+    marginRight: theme.spacing(2),
+    display: "flex",
+    "&:last-child": {
+      marginRight: 0,
+    },
+  },
+}));
+
+interface ApplicationCountsProps {
+  onClick: (kind: ApplicationKind) => void;
+}
+
+export const ApplicationCounts: FC<ApplicationCountsProps> = memo(
+  function ApplicationCounts({ onClick }) {
+    const classes = useStyles();
+    const counts = useSelector<
+      AppState,
+      Record<string, Record<string, number>>
+    >((state) => state.applicationCounts.counts);
+
+    return (
+      <div className={classes.root}>
+        {Object.keys(counts).map((kindName) => {
+          if (
+            counts[kindName][
+              APPLICATION_ACTIVE_STATUS_NAME[ApplicationActiveStatus.ENABLED]
+            ] === 0 &&
+            counts[kindName][
+              APPLICATION_ACTIVE_STATUS_NAME[ApplicationActiveStatus.DISABLED]
+            ] === 0
+          ) {
+            return null;
+          }
+
+          return (
+            <ApplicationCount
+              key={kindName}
+              kind={kindName}
+              totalCount={
+                counts[kindName][
+                  APPLICATION_ACTIVE_STATUS_NAME[
+                    ApplicationActiveStatus.ENABLED
```

Comment:
The total number should be `ENABLED` + `DISALBED` + `DELETED`.

Reply:
I think `DELETED` should not be count here 🤔  (agree with it's not just `ENABLED` but `ENABLED` + `DISABLED`)

Reply:
ah. You are right. enabled + disabled

Reply:
In the second thought, I guess if we make this "total apps" here as `ENABLED` + `DISABLED`, the below Lambda will be `2/2 apps`. 👀 
<img width="159" alt="Screen Shot 2021-03-18 at 14 56 43" src="https://user-images.githubusercontent.com/32532742/111579906-2955e180-87fa-11eb-9698-ca7f21212f10.png">
IMO, I think the current `0/2apps` pattern looks better to understand, how do you think about it? 🤔

Reply:
If I look at `2/2`, it will look like there are two active applications, so I think the `enabled/disabled` display will be easier to understand.

Reply:
You're right 👍 please go ahead with `enabled/disabled` pattern 🙆‍♀️